Maths/Fraction.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @function fraction
+ * @description This function returns the fraction of a float type number.
+ * @param {number} num - a float type number is expected, but an integer will also work.
+ * @param {number} accuracy - the accuracy of the fraction, the default is 6. like if the accuracy is 2 then for 1.333 result will be 10/9, where for 6 it will return 1333/1000.
+ * @return {Array} - an array containing the numerator and denominator of the fraction.
+ * @see https://en.wikipedia.org/wiki/Repeating_decimal and 
+ * @see https://en.wikipedia.org/wiki/Decimal#Decimal_fractions
+ * @example fraction(0.5) // [1, 2]
+ * @example fraction(0.3333333333333333) // [1, 3]
```

Comment:
Are all these examples really useful? `0.5` and `0.25` for example seem redundant.